composer.json

```diff
@@ -10,18 +10,24 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
-        "illuminate/view": "^5.5|^6.0|^7.0|^8.0"
```

Comment:
Is it possible to keep backwards compatibility with previous versions as well?

Reply:
I would  tag this as a new major version allowing v9 and v10, because all older Laravel versions (and also the illuminate packages) are already end of life. https://endoflife.date/laravel What do you think?